README.md

```diff
@@ -99,7 +99,7 @@ $ ./run start
 $ yarn hot
 ```
 
-This will launch a webpack server and nodemon app server for hot reloading any changes you make.
+This will launch a webpack server and nodemon app server for hot reloading any changes you make. NOTE: You may need to migrate and reindex. PITFALL: Make sure to run the blank-slate script prior to migrate and reindex.
```

Comment:
This section's parent section, that is about how to set up the development environment already states that you need to run the blank-state script during the process. I think this new statement may be misleading since blank-state runs are only needing if you are starting a fresh set-up. In other case you may be clearing useful debugging/testing data.
Regarding the note about migrate and reindex, Uwazi will usually write a message in the console if migrate and reindex is needed. 
IMHO we should not add this notes in here but, @RafaPolit ,maybe we should start a troubleshooting section at the end of the README with this kind of hints?

Reply:
I'll begin the troubleshooting section at the bottom

Reply:
does the new troubleshooting section look better?